snap/info.go

```diff
@@ -173,6 +173,7 @@ type Ref struct {
 	Version     string          `json:"version"`
 	Channel     string          `json:"channel"`
 	Epoch       string          `json:"epoch"`
+	Size        int64           `json:"binary_filesize"`
```

Comment:
This is a bit magic indeed, its set here https://github.com/snapcore/snapd/blob/master/store/store.go#L902 and decoded directly from the `clickindex:package` data from the store (along with anything else that comes in the snap.Ref struct). Given that we usually don't decode directly into the exported structs, maybe this needs to be rethought?